docs/docs/tooljet_database.md

```diff
@@ -149,68 +151,59 @@ A drawer from the right will open up from where you can create a new column by e
 
 </div>
 
-### Filter
-
-You can add as many filter as you want into the table by clicking on the **Filter** button present on the top of the database editor.
-
-#### Adding a filter on the table data
-- Select a **column** from the Columns dropdown
-- Choose an **[operation](#available-operations-are)**
-- Enter a **value** for the selected operation
+### Delete column
 
-#### Available operations are:
-- **equals**
-- **greater than**
-- **greater than or equal**
-- **less than**
-- **less than or equal**
-- **not equal**
-- **like**
-- **ilike**
-- **match**
-- **imatch**
-- **in**
-- **contains**
-- **contained**
-- **not**
+To delete a particular column, just click on the column header and the **delete** button will appear, click on it to delete the column.
 
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/v2-beta/database/newui/filter2.png" alt="ToolJet Database editor" />
+<img className="screenshot-full" src="/img/v2-beta/database/newui2/deletecol.png" alt="ToolJet Database editor" />
 
 </div>
 
-### Sort
+### Add new row
 
-To sort the table data, click on the **Sort** button on top, select a **column** from the dropdown, and then choose an order **ascending** or **descending**.
+To add a new row to the existing table data, click on the **Add new row** button. A drawer will open from the right where you can **enter the values** for the new row.
 
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/v2-beta/database/newui/sort2.png" alt="ToolJet Database editor" />
+<img className="screenshot-full" src="/img/v2-beta/database/newui2/addnewrow.png" alt="ToolJet Database editor" />
 
 </div>
 
-### Add new row
+### Edit row
 
-To add a new row to the existing table data, click on the **Add new row** button. A drawer will open from the right where you can **enter the values** for the new row.
+To edit the rows from the ToolJet database dashboard, click on the **Edit row** button. A drawer will open from the right from where first you need to **select the id** of the row to be edited from the dropdown and then you can edit the cell values of the selected row.
 
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/v2-beta/database/newui/newrow2.png" alt="ToolJet Database editor" />
+<img className="screenshot-full" src="/img/v2-beta/database/newui2/editrow.gif" alt="ToolJet Database editor"  />
 
 </div>
 
-### Edit row
+### Bulk upload data
 
-To edit the rows from the ToolJet database dashboard, click on the **Edit row** button. A drawer will open from the right from where first you need to **select the id** of the row to be edited from the dropdown and then you can edit the cell values of the selected row.
+You can bulk upload data to the ToolJet database by clicking on the **Bulk upload data** button on the top of the database editor. On clicking the button, a drawer will open from the right from where you can upload a **CSV** file.
```

Comment:
```suggestion
You can bulk upload data to the ToolJet database by clicking on the **Bulk upload data** button on the top of the database editor. On clicking the button, a drawer will open from the right from where you can upload a **CSV** file. This file is used to upsert records onto the table. If data for id column is missing it will insert new record with the row data else if id is present it will update the corresponding record with the corresponding row data.
```